content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: One Year of Excalidraw
+date: 2020-12-25
+author: Excalidraw Team
+link: https://github.com/orgs/excalidraw/people
+---
+
+> It's been kind of a different year, but it was the first year and pretty amazing for [Excalidraw](https://excalidraw.com).
+
+<!-- end -->
+
+Some words from the big boss [@vjeux](https://twitter.com/vjeux)
+
+> Bla bla bla
+
+## Let's start with some numbers
+
+It's been a nice and steady growth, given that our only marketing is Twitter, and of course all the love that we are receiving over there. We are reaching around **60K vistis** per month and the grand total of the visits in our first year is well over **500K**!
+
+### Visists
+
+![Weekly visists](ga-users.png)
+
+### Scenes
+
+We are storing all the scenes in our backend with [end to end encryption](/end-to-end-encryption/) (which means we don't have access to the actual drawings since they are all encrypted) and the grand total number of the scenes is almost **50K**!
```

Comment:
This looks like the same graph as the visits one. Probably not worth to duplicate it.